templates/success.tmpl

```diff
@@ -0,0 +1,76 @@
+<html>
+<head>
```

Comment:
While I'd like to see this content not repeated across different templates, it looks like Gin doesn't currently support that out of the box: https://github.com/gin-gonic/gin#html-rendering

If that's something you're willing to take on as part of this PR that would be awesome, but it's not something that I'll block merging it because of.

Reply:
+1 Templating would be really cool, but not the end of the world if we don't have it I suppose.  If it doesn't get added in this PR it should be submitted as a project issue once this is merged.